[2025-10-20T13:04:32.986+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-20T13:04:37.051+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_reference_etl.fetch_task scheduled__2025-10-19T00:00:00+00:00 [queued]>
[2025-10-20T13:04:38.430+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_reference_etl.fetch_task scheduled__2025-10-19T00:00:00+00:00 [queued]>
[2025-10-20T13:04:38.431+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-20T13:04:40.064+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): fetch_task> on 2025-10-19 00:00:00+00:00
[2025-10-20T13:04:40.099+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'stock_reference_etl', 'fetch_task', 'scheduled__2025-10-19T00:00:00+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/stock_etl_dag.py', '--cfg-path', '/tmp/tmpjtzusyf_']
[2025-10-20T13:04:40.109+0000] {standard_task_runner.py:91} INFO - Job 28: Subtask fetch_task
[2025-10-20T13:04:40.124+0000] {logging_mixin.py:188} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=579) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-10-20T13:04:40.126+0000] {standard_task_runner.py:64} INFO - Started process 587 to run task
[2025-10-20T13:04:46.703+0000] {local_task_job_runner.py:124} ERROR - Received SIGTERM. Terminating subprocesses
[2025-10-20T13:04:47.020+0000] {process_utils.py:132} INFO - Sending 15 to group 587. PIDs of all processes in the group: [587]
[2025-10-20T13:04:47.021+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 587
[2025-10-20T13:04:47.386+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=587, status='terminated', exitcode=<Negsignal.SIGTERM: -15>, started='13:04:39') (587) terminated with exit code -15
[2025-10-20T13:04:47.389+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 143
[2025-10-20T13:04:51.059+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-20T13:04:54.811+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
